### PR TITLE
Update to Freedesktop SDK 18.08

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

**Comment by nanonyme** [comment]
Remove the automatic Python3 conversion, it can generate programs that break in subtle ways

**Comment by TingPing** [comment]
> I thought gyp was added together with newer nss, it might be removable

Indeed.

File: com.valvesoftware.Steam.json [hunk 1]
```diff
@@ -20,9 +20,12 @@
         "--allow=multiarch",
         "--allow=devel",
         "--persist=.",
-        "--extension=org.freedesktop.Platform.Compat32=directory=lib/32bit",
-        "--extension=org.freedesktop.Platform.Compat32=version=1.6",
-        "--extension=org.freedesktop.Platform.GL32=directory=lib/32bit/lib/GL",
+        "--extension=org.freedesktop.Platform.Compat.i386=directory=lib/i386-linux-gnu",
+        "--extension=org.freedesktop.Platform.Compat.i386=version=unstable",
```
Comment (nanonyme) [inline com.valvesoftware.Steam.json]: Is this intentionally unstable?

Comment (valentindavid) [reply com.valvesoftware.Steam.json]: That is a mistake. Nice you saw it. I had unstable also installed.

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
Looks good to me. This should also suppress one source of confusion people have had with the app where they run into problem X and think it has something to do with dpkg

**Pull request merged**